experiments/AMIP/moist_mpi_earth/cli_options.jl

```diff
@@ -370,3 +374,6 @@ function time_to_seconds(s::String)
     end
     error("Uncaught case in computing time from given string.")
 end
+
+import Base.joinpath
+Base.joinpath(s::String, ::Nothing) = s
```

Comment:
why is this needed? generally not a good idea to type-pirate Base functions.

Reply:
Hehehe! 🏴‍☠️ This was to enable the run name cli arg to be `nothing` by default, but if this is not the best practice, I can make the default a string.

Reply:
yes, that would be a better idea.